src/Midi.js

```diff
@@ -161,6 +161,21 @@ class Midi {
 	}
 
 	/**
+	 * Convert JSON to Midi object
+	 * @param {object} json
+	 * @returns {Midi}
+	 */
+	fromJSON(json){
```

Comment:
I would make this a static method.  (`encode` probably should be too, but that's a separate conversation).